thirdparty/lua-serialize/CMakeLists.txt

```diff
@@ -12,7 +12,9 @@ assert_var_defined(OUTPUT_DIR)
 
 ep_get_source_dir(SOURCE_DIR)
 
-set(BUILD_CMD sh -c "${KO_MAKE_RECURSIVE} -j${PARALLEL_JOBS} CC=\"${CC}\" LDFLAGS=\"${LDFLAGS}\" OUTPUT_DIR=${OUTPUT_DIR}")
+set(PATCH_CMD sh -c "patch -N -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/lua-serialize-lto.patch || true")
```

Comment:
Btw, not directly related to this PR but I've been wondering why let the patches fail semi-quietly? That is, you won't notice without studying the logs.

Reply:
Wasn't that to handle rebuilds? I have no idea if that still makes sense with the CMake system, though :D.

Reply:
Afaik CMake handles when to patch or not to patch with an uncharacteristic lack of stupidity.

Reply:
Then yeah, it doesn't make sense anymore, and that's just a bad habit carried over from the old makefile ;)

Reply:
In that case I'll submit a PR to get rid of the load of 'em. It's tripped me up more than once.

Reply:
Most recently https://github.com/koreader/koreader-base/pull/860

Reply:
Oh, you can see I already got rid of the `|| true` there, so you'd have noticed if that were somehow trying to apply itself multiple times. ;-)